src/azure-cli/azure/cli/command_modules/search/aaz/latest/search/service/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "search service",
+)
+class __CMDGroup(AAZCommandGroup):
+    """Demo
```

Comment:
Please update the help message of this command group.

Reply:
done